lightspeed/modules/proc_create-connection-secrets.adoc

```diff
@@ -4,7 +4,7 @@
 
 = Creating connection secrets
 
-Use this procedure to create secrets to connect to {PlatformName} and {ibmwatsonxcodeassistant}.
+You must create authorization secret to connect to {PlatformName}, and a model secret to connect to {ibmwatsonxcodeassistant}. If you need to trust a custom Certificate Authority, you must create a bundle secret.
```

Comment:
Would change to "You must create *an* authorization secret"

Reply:
Updated